src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -231,7 +243,24 @@ class IrsPlan extends React.Component<
       this.setState({ newPlan: planById });
     }
 
-    const otherJurisdictionSupersetParams = { row_limit: 10000 };
+    // Get assignments
+    if (planId) {
+      // fetch all organizations
+      OpenSRPOrganizationService.list()
+        .then((response: Organization[]) => {
+          // save all organizations to store
+          store.dispatch(fetchOrganizationsActionCreator(response));
+          // get all assignments
+          return this.getAllAssignments(planId, response, (nextAssignments: Assignment[]) =>
+            store.dispatch(fetchAssignmentsActionCreator(nextAssignments))
+          );
+        })
+        .catch((err: Error) => {
+          /** TODO - find something to do with error */
+        });
```

Comment:
Fetch initial organizations and subsequently fetch assignments in `getAllAssignments`.